capa/rules/__init__.py

```diff
@@ -194,6 +196,59 @@ def __repr__(self):
         return str(self)
 
 
+# COM data source https://github.com/stevemk14ebr/COM-Code-Helper/tree/master
+VALID_COM_TYPES = {
+    "class": {"db_path": "assets/classes.json.gz", "prefix": "CLSID_"},
+    "interface": {"db_path": "assets/interfaces.json.gz", "prefix": "IID_"},
+}
+
+
+def translate_com_feature(com_name: str, com_type: str) -> ceng.Or:
+    if com_type not in VALID_COM_TYPES:
+        raise InvalidRule(f"Invalid COM type present {com_type}")
+
+    CD = Path(__file__).resolve().parent.parent.parent
```

Comment:
we'll need to check that this works with pyinstaller binaries. do you happen to know @Aayush-Goel-04?

Reply:
recommend using `get_default_root` from here to be sure: https://github.com/mandiant/capa/blob/77de088ac90d97d736091c43474eb2cf41711a3d/capa/main.py#L444